src/Analysis/Engine/Impl/ModuleAnalysis.cs

```diff
@@ -251,6 +270,18 @@ private class DefinitionWalker : PythonWalkerWithLocation {
             return new VariablesResult(variables, unit.Tree);
         }
 
+        private bool IsFunctionArgument(IScope scope, IAnalysisVariable v) {
+            if (v.Variable?.Types?.MaybeEnumerate().FirstOrDefault() is ParameterInfo) {
```

Comment:
Can `v.Variable` actually be `null`?

Reply:
I am not 100% sure. Original code did `if (referenceable is VariableDef def) { `. I think is is *always* a variable but I'd rather be safe.